.changeset/mighty-things-brake.md

```diff
@@ -0,0 +1,6 @@
+---
+"@digdir/designsystemet-react": patch
+"@digdir/designsystemet-css": patch
```

Comment:
```suggestion
```

Reply:
We have changed the css for details so I don't see why we should remove this from changeset

Reply:
Because the dependency is not used in our css package

Reply:
ah, true. Will make a separate changeset. 👍